com.unity.ml-agents.extensions/Editor/GridSensorComponentEditor.cs

```diff
@@ -82,14 +64,21 @@ public override void OnInspectorGUI()
             }
             EditorGUI.EndDisabledGroup();
             EditorGUILayout.PropertyField(so.FindProperty(nameof(GridSensorComponent.m_CompressionType)), true);
+            EditorGUI.BeginDisabledGroup(!EditorUtilities.CanUpdateModelProperties());
```

Comment:
nit: don't think these need to be disabled during play mode, since they don't affect observation size.

Reply:
updated.